pkgs/by-name/ag/age-plugin-tpm/package.nix

```diff
@@ -39,10 +43,15 @@ buildGoModule rec {
     mainProgram = "age-plugin-tpm";
     homepage = "https://github.com/Foxboron/age-plugin-tpm";
     license = licenses.mit;
-    platforms = platforms.linux;
+    platforms = platforms.all;
     maintainers = with maintainers; [
       kranzes
       sgo
     ];
   };
+
+  passthru.tests = {
```

Comment:
meta goes always after passthru.

Reply:
Done! I was curious if there was a style guide for this kind of stuff? Also wondering about the sort order of callPackage args, if there's a style that's preferred. Thanks!